src/components/Breadcrumbs/Breadcrumbs.mdx

```diff
@@ -69,6 +70,31 @@ the truncated page title.
 
 <Canvas of={BreadcrumbsStories.LongText} />
 
+## Custom Link and Link Props
+
+The `customLink` prop allows apps with internal routers (Next.js, for example) to pass their custom link component into Breadcrumbs so the links can use client-side routing.
+To maximize these benefits, the `linkProps` prop on `breadcrumbsData` allows accompanying custom props on a link-by-link basis.
+
+Note: assigning a value for `href` in `linkProps` will override the `url` prop, so avoid this.
+
+<Source
+  code={`
+  // Example with Next.js Link and associated props.
+const breadcrumbsData = [
+  { url: "#", text: "Parent", linkProps: { replace: true, scroll: false }}
+  { url: "#", text: "Child", , linkProps: { replace: true }}
+  { url: "#", text: "Grandchild" },
+];
+// ...
+<Breadcrumbs customLink={Link} breadcrumbsData={breadcrumbsData} />
```

Comment:
At the top of this code example snippet, add the import for the `Link` component from next/link, to explicitly show where it's imported from.